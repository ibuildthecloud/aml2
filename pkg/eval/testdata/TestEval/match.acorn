Test: schema {
	match ".*": 2
	a: number
	missing: string
}

let x: 1

{
	Test
	{
		blah: 5
		f: "hi"
		a: 3
		y: x
		missing: "nope it's here"
		blah2: blah
	}
}
