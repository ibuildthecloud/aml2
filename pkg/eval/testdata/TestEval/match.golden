{
  "a": 3,
  "f": 2,
  "missing": 2
}